react/typescript/looks-query/src/index.tsx

```diff
@@ -32,18 +32,9 @@ window.addEventListener("DOMContentLoaded", async (event) => {
   const root = document.createElement("div");
```

Comment:
semicolon again? Guess that's not a prettier setting in this repo

Reply:
Yep - on the list